jupyter_console/ptshell.py

```diff
@@ -747,9 +748,15 @@ def handle_image_tempfile(self, data, mime):
             fmt = dict(file=f.name, format=imageformat)
             args = [s.format(**fmt) for s in self.tempfile_image_handler]
             subprocess.call(args, stdout=devnull, stderr=devnull)
+        return True
```

Comment:
this proc can also fail. but we can use the returncode returned by `subprocess.call`